### DISCUSSION TITLE
可以直接打开mod列表里面的模组的pcl下载界面

### DISCUSSION THREAD

**Discussion by qizri** [Meloong-Git/PCL#1335] [提问] [answered]
或者直接打开pcl的搜索页面，就像直接打开百科搜索页面一样。这样就比较好更新模组或者下载其他版本的模组。

**Comment by Chiloven945** [comment]
这里是提问，有新功能提议交issues哈

**Reply by qizri** [reply]
（挠头.jpg）我还以为直接点就可以提交建议呢,不太会用，现在我找不到删除的地方了。

**Reply by Chiloven945** [reply]
没事，这里没大问题，但是我估计这个功能龙猫不大会同意因为有点难实现，你试着交一下吧

**Comment by WForst-Breeze** [comment, marked as answer]
反馈者已提交新功能提案，后续讨论请移步 Issues #1336~
